R/Spectra-functions.R

```diff
@@ -484,3 +484,66 @@ combineSpectra <- function(x, f = x$dataStorage, p = x$dataStorage,
         else common(y, z, tolerance = tolerance, ppm = ppm)
     })
 }
+
+#' @export
+#'
+#' @rdname Spectra
+spectraVariableMapping <- function(format = c("mgf")) {
```

Comment:
Isn't there something like this is `MsBackendMgf`? Anyway, that mapping should be defined only once, and `Spectra` is the only package for that. it will this be important (note to self) to reuse it in `MsBackendMgf`.

Reply:
Agree. At first glance I haven't seen anything in `MsBackendMgf`.